builtin/providers/google/resource_compute_address.go

```diff
@@ -13,37 +13,51 @@ import (
 func resourceComputeAddress() *schema.Resource {
 	return &schema.Resource{
 		Create: resourceComputeAddressCreate,
-		Read:   resourceComputeAddressRead,
+		Read:	resourceComputeAddressRead,
 		Delete: resourceComputeAddressDelete,
 
 		Schema: map[string]*schema.Schema{
 			"name": &schema.Schema{
-				Type:     schema.TypeString,
+				Type:	  schema.TypeString,
 				Required: true,
 				ForceNew: true,
 			},
 
 			"address": &schema.Schema{
-				Type:     schema.TypeString,
+				Type:	  schema.TypeString,
 				Computed: true,
 			},
 
 			"self_link": &schema.Schema{
-				Type:     schema.TypeString,
+				Type:	  schema.TypeString,
 				Computed: true,
 			},
+
+			"region": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
 		},
 	}
 }
 
+func getOptionalRegion(d *schema.ResourceData, config *Config) string {
```

Comment:
I wish we could just use this as `DefaultFunc`, but because of that dependency (`*Config`), it is effectively impossible. :-/